src/style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  margin: 0;
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
+  .three {
+    order: 2;
+  }
+
+  .four {
+    order: 1;
+  }
+
+  .five {
+    order: 3;
+  }
+
+  .six {
+    order: 4;
+  }
+}
+
+@media (max-width: 599px) {
+  .four {
+    order: 1;
+  }
+
+  .five {
+    order: 2;
+  }
+
+  .six {
+    order: 3;
+  }
+}
+
+@media (min-width: 900px) {
+  .wrap {
+    max-width: 1199px;
```

Comment:
don't restrict container width, it should work for big resolutions too, try to work around flex-basis